.github/workflows/test-import.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
           cd $(mktemp -d)
           echo "$GO_MAIN" > main.go
           go mod init github.com/zarf-dev/test-import
-          go mod edit -replace github.com/zarf-dev/zarf=github.com/${{ github.repository }}@${COMMIT_SHA:0:12}
+          go get github.com/${{ github.event.pull_request.head.repo.full_name }}@$COMMIT_SHA
```

Comment:
This should in my head still use `go mod edit` to replace the original Zarf package with the fork.